core/domain/Domain.php

```diff
@@ -20,7 +20,9 @@ class Domain extends DomainBase implements CaffeinatedInterface
      */
     const API_NAMESPACE = 'ee/v';
 
-    const ASSET_NAMESPACE = 'eventespresso-';
+    const ASSET_NAMESPACE = 'eventespresso';
+
+    const TEXT_DOMAIN = 'event_espresso';
```

Comment:
this is NOT used within any WP i18n functions and only for secondary text domain identification.